services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler.py

```diff
@@ -430,6 +459,32 @@ async def restart_containers(self, node_uuid: NodeID) -> None:
     def _enqueue_observation_from_service_name(self, service_name: str) -> None:
         self._trigger_observation_queue.put_nowait(service_name)
 
+    def __get_observation_task(
+        self,
+        dynamic_sidecar_settings: DynamicSidecarSettings,
+        dynamic_scheduler: DynamicServicesSchedulerSettings,
+        service_name: str,
+    ) -> asyncio.Task:
+        scheduler_data: SchedulerData = self._to_observe[service_name]
+        observation_task = asyncio.create_task(
+            observing_single_service(
+                scheduler=self,
+                service_name=service_name,
+                scheduler_data=scheduler_data,
+                dynamic_sidecar_settings=dynamic_sidecar_settings,
+                dynamic_scheduler=dynamic_scheduler,
+            ),
+            name=f"observe_{service_name}",
```

Comment:
MINOR: we use to add a prefix with the name of the module to identify where it was created, e.g.
```python
name=f"{__name__}.observe_{service_name}",
```